styleguide/pane.scss

```diff
@@ -7,6 +7,11 @@
 $pane-margin: 30vh;
 $easeOutExpo: cubic-bezier(.190, 1.000, .220, 1.000);
 
+.hide,
+.kiln-pane-template-elements {
+  display: none !important; // Always hide your skeletons and other hidden things
```

Comment:
hmm, I could go either way on this. When is it necessary to use `!important`? I would feel better not using until it reveals itself as being necessary.